.github/workflows/playwright.yml

```diff
@@ -1,8 +1,18 @@
 name: Playwright Tests
 on:
     schedule:
-        # Firefox tests running on Monday, Wednesday and Friday. Chrome runs Tuesday and Thursday.
-        - cron: '0 5 * * 1,2,3,4,5'
+        # Playwright tests are running automatically in Firefox on each Monday & in Chrome on each Friday.
+        - cron: '0 5 * * 1,5'
+    workflow_dispatch:
+        inputs:
+            Browsers:
+                description: 'Browsers'
+                required: true
+                default: 'firefox'
```

Comment:
Nit. I don't think you need the single quotes around `'Browsers'` and `'firefox'`?